wandb/apis/public.py

```diff
@@ -2092,6 +2188,80 @@ def download_file(name):
             termlog('Done. %.1fs' % (time.time() - start_time), prefix=False)
         return dirpath
 
+    def file(self, root=None, name=None):
```

Comment:
Passing name to this duplicates functionality in get_path() stuff. Do you think we should have both?

It might make sense to just remove the name parameter here. If you want the other behavior you use get_path().local().